### PR TITLE
[TUNA] FIN subtree updates JSON versioning

### PR DESCRIPTION
**Pull request by alexandraBara**
Pulled in the most recent version of FIN https://github.com/ROCmSoftwarePlatform/fin:
- contains JSON versioning

### CODE REVIEW

File: fin/src/include/conv_fin.hpp [hunk 1]
```diff
@@ -994,17 +994,25 @@ int ConvFin<Tgpu, Tref>::ProcessStep(const std::string& step_name)
     if(step_name == "copy_buf_from_device")
         return CopyFromDevice();
     if(step_name == "applicability")
+    {
```
Comment (junliume) [inline fin/src/include/conv_fin.hpp:997]: @alexandraBara  why this change? Or why not change them all? :)

**Review by JehandadKhan — APPROVED** [review]

**Comment by JehandadKhan** [comment]
@alexandraBara Please check the CI

**Comment by junliume** [comment]
It should not have impacted CI while the pipeline keep stuck for other reasons. Restarted again and let's monitor it.

**Review by junliume — APPROVED** [review]
